src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -16,6 +16,33 @@ public Rectangle(Integer width, Integer length, String color) {
         this.color = color;
     }
 
+    @Override
+    public int hashCode() {
+        int result = 0;
+        result = color.length() + width.intValue() + length.intValue();
```

Comment:
is it ok to use length of fields to find hashcode?

Reply:
Actually I don't know for sure but it has worked in this video https://youtu.be/Lk2vF7xz8tE?t=667

Reply:
which hash code we will have if we create next objects:
Rectangle rec1 = new Rectangle(20, 25, "green");
Rectangle rec2 = new Rectangle(20, 25, "black");
rec1.hachCode()   ?   rec2.hachCode() 

our hachCode() shoud be different

Reply:
after Q&A session somehow my latest commit didn't get to GitHub. Now that's gonna be ok.